### ISSUE TITLE
Discord-Image-Classification-Bot

### ISSUE THREAD

**Issue by ksushanovl-lab** [ksushanovl-lab/Discord-Image-Classification-Bot#1]
import tf_keras as keras
from tf_keras.models import load_model  
from PIL import Image, ImageOps
import numpy as np

class PatchedDepthwiseConv2D(keras.layers.DepthwiseConv2D):
    """DepthwiseConv2D, игнорирующий несовместимый параметр 'groups'
    (частая проблема моделей Teachable Machine при загрузке в новых
    версиях Keras)."""
    def __init__(self, *args, **kwargs):
        kwargs.pop("groups", None)
        super().__init__(*args, **kwargs)
        
def get_class(model_path, labels_path, image_path):
    np.set_printoptions(suppress=True)
    model = load_model(
        model_path,
        compile=False,
        custom_objects={"DepthwiseConv2D": PatchedDepthwiseConv2D},
    )

    class_names = open(labels_path, "r", encoding="utf-8").readlines()
    data = np.ndarray(shape=(1, 224, 224, 3), dtype=np.float32)
    image = Image.open(image_path).convert("RGB") 
    size = (224, 224)
    image = Image0ps.fit(image, size, Image. Resampling. LANCZOS)
    image_array = пp.asarray(image)
    normalized_image_array = (image_array.astype(np.float32) / 127.0) - 1
    data[0] = normalized_image_array
    prediction = model.predict(data)
    index = np.argmax(prediction)
    class_name = class_names[index]
    confidence_score = prediction[0] [index]
    return class_name, confidence_score

if __name__ == "__main__":
    name, score = get_class("keras_model.h5", "labels.txt", "проба.png")
    print(f"Class: {name}, Confidence: {score}")

**Comment by ksushanovl-lab** [comment]
бот с разными функциями
